rpcs3/Emu/RSX/Overlays/overlay_controls.h

```diff
@@ -468,15 +468,15 @@ namespace rsx
 				is_compiled = false;
 			}
 
-			void set_text(const std::u32string& text)
+			virtual void set_wide_text(const std::u32string& text)
```

Comment:
calling unicode strings wide is a bit old fashioned, maybe we can call it something else?
set_unicode_text() or set_text_unicode() or something.
std::wstring exists that literally means wide string, but that is not what we're using here, we're explicitly using utf-32.

Reply:
fixed